ci/tests/puppeteer/scenarios/passwordless-login-with-reset-password/script.json

```diff
@@ -0,0 +1,33 @@
+{
+  "dependencies": "passwordless-webflow",
+
```

Comment:
Please add a condition here to require docker.

Reply:
Sure.